test/sma.test.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const { expect } = require('chai')
+const SMA = require('../src/sma')
+
+module.exports = () => {
+  describe('# sma-tests', () => {
+
+    it('SMA - it should fail with non numeric values', () => {
+      const values = ['10161', 'a10261.235']
+
+      return expect(
+        SMA.bind(null, values)
+      ).to.throw()
+    })
+
+    it('SMA - it should fail with no values', () => {
+      return expect(
+        SMA.bind(null, [])
+      ).to.throw('ERR_NO_VALUES_PROVIDED')
+    })
+
+    it('SMA - it should match the expected value', () => {
+      const values = ['10161', '10261.235', '10324.2567', 10724, '10232', '10510.25847', '10261.826']
```

Comment:
hm, no timeframe supported, i guess this is on the service which implements the lib, can you confirm?